hoomd/operations.py

```diff
@@ -65,6 +141,7 @@ def schedule(self):
         self._scheduled = True
 
     def unschedule(self):
+        """Undo the effects of `Operations.schedule`."""
```

Comment:
When would a user want to use this method? A little more documentation would be helpful.

Reply:
A user in general should not have to deal with this especially if we use `sim.run(0)`.